src/main/resources/add-ons-to-index.json

```diff
@@ -328,6 +340,9 @@
       "type": "OMOD",
       "name": "Birt Reports",
       "status": "INACTIVE",
+	  "tags": [
+        "reporting"
```

Comment:
this tag is fine, though it's probably not so valuable to tag inactive or deprecated addons.